### PR TITLE
Pull Request v3 issue #305

### PR DESCRIPTION
**Pull request by abhishek58g**
Pull Request v3 issue #305

### CODE REVIEW

File: tests/Test.ADAL.NET/AuthenticationContextProxy.cs [hunk 1]
```diff
@@ -168,9 +168,9 @@ public void SetCorrelationId(Guid correlationId)
             try
             {
                 AuthenticationResult result = null;
-                using (Timer abortTest = new Timer(10 * 1000)) // 10 seconds for test execution
+                using (Timer abortTest = new Timer(1000 * 1000)) // 10 seconds for test execution
```
Comment (kpanwar) [inline tests/Test.ADAL.NET/AuthenticationContextProxy.cs]: revert this change

File: tests/Test.ADAL.NET/AuthenticationContextProxy.cs [hunk 2]
```diff
@@ -168,9 +168,9 @@ public void SetCorrelationId(Guid correlationId)
...
                 {
-                    using (Timer uiSupply = new Timer(1500))
+                    using (Timer uiSupply = new Timer(150000))
```
Comment (kpanwar) [inline tests/Test.ADAL.NET/AuthenticationContextProxy.cs]: revert this change

File: tests/Test.ADAL.NET/AuthenticationContextProxy.cs [hunk 3]
```diff
@@ -231,9 +231,9 @@ public void SetCorrelationId(Guid correlationId)
 
             try
             {
-                using (Timer abortTest = new Timer(10 * 1000)) // 10 seconds for test execution
+                using (Timer abortTest = new Timer(10 * 10000)) // 10 seconds for test execution
```
Comment (kpanwar) [inline tests/Test.ADAL.NET/AuthenticationContextProxy.cs]: revert this change

File: tests/Test.ADAL.NET/AuthenticationContextProxy.cs [hunk 4]
```diff
@@ -231,9 +231,9 @@ public void SetCorrelationId(Guid correlationId)
...
                 {
-                    using (Timer uiSupply = new Timer(1500))
+                    using (Timer uiSupply = new Timer(150000))
```
Comment (kpanwar) [inline tests/Test.ADAL.NET/AuthenticationContextProxy.cs]: revert this change

**Comment by abhishek58g** [comment]
I have reverted back the time changes for commits.
